src/repository/repository.reducer.js

```diff
@@ -62,19 +63,21 @@ export const repositoryReducer = (state = initialState, action = {}) => {
       return {
         ...state,
         issues: [],
+        error: '',
         isPendingRepository: true,
       };
     case GET_REPOSITORY.SUCCESS:
       return {
         ...state,
         repository: action.payload,
+        hasRepoExist: true,
+        error: '',
         isPendingRepository: false,
       };
     case GET_REPOSITORY.ERROR:
       return {
-        ...state,
+        ...initialState,
         error: action.payload,
-        isPendingRepository: false,
```

Comment:
I just noticed that this line was removed. Do you want to keep pending state when an error occurs?

Reply:
@chinesedfan it's because we take `...initialState` that already had `isPendingRepository` state as `false`.